ansible/roles/csm.rr.mgmt_nodes_placement_validation/files/mgmt_nodes_placement_validation.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python3
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+import json
+import subprocess
+import re
+import sys
+
+min_rack_cnt = 3
+min_master_node_cnt = 3
+min_worker_node_cnt = 3
+min_storage_node_cnt = 3
+
+class bcolors:
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    UNDERLINE = '\033[4m'
+
+def get_node_rack_missing_quorum(rack_cnt):
+    return (rack_cnt - min_rack_cnt)
+
+def validate_master_nodes_placement(placements_dict):
+    rack_cnt=1
+    master_nodes_cnt=0
+    missing_cnt=0
+    print(bcolors.OKGREEN + bcolors.UNDERLINE + "\nValidate Management Master nodes"+ bcolors.ENDC)
```

Comment:
Will these colors be preserved in the Kubernetes pod logs? If not, may be better to omit them, otherwise they will make the output harder to parse.